docs/pages/material-ui/api/popper.json

```diff
@@ -67,7 +67,14 @@
     "import Popper from '@mui/material/Popper';",
     "import { Popper } from '@mui/material';"
   ],
-  "classes": [],
+  "classes": [
+    {
+      "key": "root",
```

Comment:
Why didn't it have a class name before?

Reply:
It was coming from Base UI, I think the docs didn't picked it up.